cmd/jujud/agent/unit.go

```diff
@@ -224,3 +249,8 @@ func (a *UnitAgent) validateMigration(apiCaller base.APICaller) error {
 	}
 	return nil
 }
+
+// SetStatus implements upgradesteps.StatusSetter
```

Comment:
Rather than cluttering up this type with more methods, can you please make a separate "nopStatusSetter" type with this method? and then just `return nopStatusSetter{}, nil` in the function set on the manifold config. That'll make it a bit more self-documenting.

Reply:
done now in unit/manifolds.go